tracer/arithmetization/src/main/java/net/consensys/linea/zktracer/module/rlpUtils/InstructionDataPricing.java

```diff
@@ -15,60 +15,39 @@
 
 package net.consensys.linea.zktracer.module.rlpUtils;
 
-import static net.consensys.linea.zktracer.Trace.LLARGE;
 import static net.consensys.linea.zktracer.Trace.RLP_UTILS_INST_DATA_PRICING;
-import static net.consensys.linea.zktracer.module.rlpUtils.WcpExoCall.callToLeq;
 
-import java.util.ArrayList;
-import java.util.List;
 
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.experimental.Accessors;
 import net.consensys.linea.zktracer.Trace;
 import net.consensys.linea.zktracer.module.rlptxn.cancun.GenericTracedValue;
-import net.consensys.linea.zktracer.module.wcp.Wcp;
 import net.consensys.linea.zktracer.types.Bytes16;
 import org.apache.tuweni.bytes.Bytes;
-import org.apache.tuweni.bytes.Bytes32;
 
 @Accessors(fluent = true)
 @EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
 public class InstructionDataPricing extends RlpUtilsCall {
-  private static final Bytes32 TWO_FIFTY_FIVE = Bytes32.leftPad(Bytes.fromHexString("0xff"));
   @EqualsAndHashCode.Include @Getter private final Bytes16 limb;
   @EqualsAndHashCode.Include @Getter private final short nBytes;
-  private final List<Short> zeros;
-  private final List<Short> nonZeros;
+  public Short zeros;
```

Comment:
These don't need to be `public`.  Its really not great practice to have `public` non-`final` fields.